src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,37 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+
 public class FileWork {
+    public static final String WORDS_SPLIT = "\\W+";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        StringBuilder stringBuilder = new StringBuilder();
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName));
```

Comment:
```suggestion
        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName)));
```
let's use try with resources, because if you close them in try block you will get problems when exception occurs (.close() method won't be called)